qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -810,10 +921,13 @@ def __init__(self, name=None, label=None, unit=None, instrument=None,
 
         self._set(value)
 
-    def get(self):
+    def get_raw(self):
```

Comment:
why is this not just `get`?
`get_raw` seems to imply some byte code or something, which it does not do.